### ISSUE TITLE
Crash on deleting text in Shop Name window

### ISSUE THREAD

**Issue by andrewcsmith** [OpenRCT2/OpenRCT2#5487]
**OS:** OSX
**Version:** 10.11.6
**Commit/Build:** baf9493

Game crashes when deleting text from the shop title window.

- [ ] Reproducible in RCT2 (vanilla)? (n/a, don't have windows)
- [ ] Multiplayer?

**Steps to reproduce:**
1. Open shop window dialog
2. Press "backspace"

**Dump file**
[dump.txt.zip](https://github.com/OpenRCT2/OpenRCT2/files/1006701/dump.txt.zip)

**Save game:**

[autosave_2017-05-16_22-58-23.sv6.zip](https://github.com/OpenRCT2/OpenRCT2/files/1006703/autosave_2017-05-16_22-58-23.sv6.zip)

**Comment by janisozaur** [comment]
```
0   libsystem_platform.dylib      	0x00007fff866c560f _platform_memmove$VARIANT$Nehalem + 175
1   website.openrct2.OpenRCT2     	0x000000010000daeb OpenRCT2::Ui::TextComposition::Delete() + 39
2   website.openrct2.OpenRCT2     	0x000000010000d9b2 OpenRCT2::Ui::TextComposition::HandleMessage(SDL_Event const*) + 560
3   website.openrct2.OpenRCT2     	0x000000010000e370 UiContext::ProcessMessages() + 584
4   website.openrct2.OpenRCT2     	0x000000010004023f OpenRCT2::Context::RunVariableFrame() + 79
5   website.openrct2.OpenRCT2     	0x00000001000401cf OpenRCT2::Context::RunGameLoop() + 111
6   website.openrct2.OpenRCT2     	0x000000010003f9cf OpenRCT2::Context::Launch() + 171
7   website.openrct2.OpenRCT2     	0x000000010003f6bc OpenRCT2::Context::RunOpenRCT2(int, char**) + 22
8   website.openrct2.OpenRCT2     	0x000000010000dc64 main + 86
9   libdyld.dylib                 	0x00007fff925df5ad start + 1
```

@andrewcsmith why are you using baf9493 and not `develop`?

**Comment by rwjuk** [comment]
Fixed in 202b069
